### PR TITLE
Revamped - state variables reorder

### PR DESCRIPTION
**Pull request by mtabasco**
Remove: OperatorFeeConfig, DAO and Network structs. Create specific storage for Protocol state variables.

### CODE REVIEW

File: contracts/interfaces/ISSVNetworkCore.sol [hunk 1]
```diff
@@ -10,7 +10,7 @@ interface ISSVNetworkCore {
     struct Snapshot {
         /// @dev The block number when the snapshot was taken
         uint32 block;
-        /// @dev The last index calculated by the formula index += (currentBlock - block) * fee
+        /// @dev The last index calculated by the formula index += (currentBlock    block) * fee
```
Comment (andrew-blox) [inline contracts/interfaces/ISSVNetworkCore.sol:13]: Why did you remove the - from the console?

File: contracts/libraries/OperatorLib.sol [hunk 2]
```diff
@@ -43,8 +44,7 @@ library OperatorLib {
                 if (!increaseValidatorCount) {
                     operator.validatorCount -= deltaValidatorCount;
                 } else if (
-                    (operator.validatorCount += deltaValidatorCount) >
-                    s.validatorsPerOperatorLimit
+                    (operator.validatorCount += deltaValidatorCount) > SSVStorageProtocol.load().validatorsPerOperatorLimit
```
Comment (lior-blox) [inline contracts/libraries/OperatorLib.sol:47]: Its loaded 4 times, isnt better to load it once?

File: contracts/modules/SSVDAO.sol [hunk 3]
```diff
@@ -1,63 +1,59 @@
...
 
-        SSVStorage.load().network.updateNetworkFee(fee);
+        SSVStorageProtocol.load().updateNetworkFee(fee);
 
         emit NetworkFeeUpdated(previousFee, fee);
     }
 
     function withdrawNetworkEarnings(uint256 amount) external override {
-        DAO memory dao_ = SSVStorage.load().dao;
+        StorageProtocol storage sn = SSVStorageProtocol.load();
```
Comment (lior-blox) [inline contracts/modules/SSVDAO.sol:28]: use proper param names

File: contracts/libraries/OperatorLib.sol (hunk 2 above)
Comment (mtabasco) [reply contracts/libraries/OperatorLib.sol:47]: loaded 4 times: 116170 gas
cached into memory variable: 116412
@andrew-blox

File: contracts/modules/SSVDAO.sol (hunk 3 above)
Comment (mtabasco) [reply contracts/modules/SSVDAO.sol:28]: done @andrew-blox

**Comment by mtabasco** [comment]
2 storage structs vs one (gas usage):

liquidate function:
2: 132700
1: 131707

register validator (7 operators):
2: 298000
1: 298100

register operator:
2: 133392
1: 133392
@andrew-blox
